pom.xml

```diff
@@ -111,6 +111,10 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-jms</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-aws</artifactId>
```

Comment:
Shouldn't we use `camel-aws-starter` instead?
https://camel.apache.org/components/2.x/aws-s3-component.html#_spring_boot_auto_configuration

Reply:
we can, but with camel-aws is working fine, we are not using any spring boot autoconfiguration
do you want me to change it ?

Reply:
I see, if the configuration management is something you already implemented in your code, i would say we already spent the effort for doing it so let's keep it as it is.